src/callbacks.rs

```diff
@@ -57,11 +63,28 @@ pub extern fn view_created(view: WlcView) -> bool {
     trace!("view_created: {:?}: \"{}\"", view, view.get_title());
     let output = view.get_output();
     if let Ok(mut tree) = tree::try_lock_tree() {
-        tree.add_view(view.clone());
-        drop(tree);
+        if tree.get_active_container().is_none() {
+            warn!("Could not create view, there is no focus \
+                    so Way-Cooler doesn't know where to put it");
```

Comment:
Very trivial complaint, but do we want way-cooler spelled like this to show up in logs?